Shared.Tests/CloudinaryTest.cs

```diff
@@ -113,9 +130,9 @@ public void TestUploadLocalVideo()
         public void TestUploadCustom()
         {
             var file = new FileDescription(m_testVideoPath);
-
             var uploadResult = m_cloudinary.Upload("video", null, file);
 
+            m_cloudinary.DeleteResources(new DelResParams { PublicIds = new List<string> { uploadResult.PublicId }, ResourceType = ResourceType.Video });
```

Comment:
Why here? move to tearDown
If you need to make sure no resources exist before test, run it on SetUp